### ISSUE TITLE
Miner not working

### ISSUE THREAD

**Issue by Mudassar5875** [UnamSanctam/SilentXMRMiner#527]
Sir miner is working as I checked it using checker file but there are no workers showing on nanopool dashboard and I am not getting profit and I also left to computer own for 4 hours but the worker is not showing on nanopool dashboard and I am not getting profit

**Comment by UnamSanctam** [comment]
Post a screenshot of what the checker returns.

**Comment by Mudassar5875** [comment]
[image: image.png]

**Comment by Mudassar5875** [comment]
![image](https://user-images.githubusercontent.com/106976496/172309610-2b2e4d76-d9d5-46a2-abae-2b6e23902d88.png)

**Comment by Mudassar5875** [comment]
bro it is showing that the silent miner is working but on pool website I am getting no profit.

**Comment by Mudassar5875** [comment]
cpu is at high usage but getting no profit and showing no workers on nanopool website

**Comment by UnamSanctam** [comment]
You do have a hashrate on the nanopool website, or are you mining to that address with something else as well?

**Comment by Mudassar5875** [comment]
Yes I am mining with the same address with nanominer as well. But When I use nanominer it gives me profit but on silent miner xmr It is not giving me profit>

**Comment by UnamSanctam** [comment]
Well first, you should disable "NiceHash Mining" and then also update to the new version, but how would you know that the miner isn't giving you hashrate? Since you're not using worker names.

**Comment by Mudassar5875** [comment]
how to use workers name

**Comment by Mudassar5875** [comment]
I am already using the latest version of silent miner

**Comment by UnamSanctam** [comment]
No that's not the latest version, but for nanopool you add the worker name after the wallet address like such `Wallet.WorkerName`

**Comment by Mudassar5875** [comment]
Can you please write a worker name for me I am not understanding how to write the worker name

**Comment by Mudassar5875** [comment]
bro I am using the latest version which is 1.5.1

**Comment by UnamSanctam** [comment]
No you're using SilentCryptoMiner 2.4.0, the newest one is 2.4.1 now https://github.com/UnamSanctam/SilentCryptoMiner/releases. But to add a worker name, in your "Wallet Address/User" field add .Worker at the end of your wallet address.

**Comment by Mudassar5875** [comment]
Thankyou sir

**Comment by Mudassar5875** [comment]
Thankyou sir now it is working fine
